cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -94,30 +123,15 @@ enum level_type {
  *
  */
 struct PageNestingInfo {
-  // input repetition/definition levels are remapped with these values
-  // into the corresponding real output nesting depths.
-  int32_t start_depth;
-  int32_t end_depth;
-
-  // set at initialization
-  int32_t max_def_level;
-  int32_t max_rep_level;
+  // set at initialization (see start_offset_output_iterator in reader_impl_preprocess.cu)
```

Comment:
Is splitting this worth the memory savings? I get that we want PageNestingDecodeInfo to be as small as possible, but is it worth it now that it's been done? I will leave this simply as a question. I don't have a good idea of how much it gained us.